parm/config/config.ufs

```diff
@@ -197,23 +197,28 @@ case "${fv3_res}" in
         export nthreads_fv3_gfs=4
         export cdmbgwd="4.0,0.05,1.0,1.0"  # mountain blocking, ogwd, cgwd, cgwd src scaling
         export WRITE_GROUP=4
-        export WRTTASK_PER_GROUP=64  # TODO: refine these numbers when a case is available
+        export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=10  # TODO: refine these numbers when a case is available
         export WRITE_GROUP_GFS=4
-        export WRTTASK_PER_GROUP_GFS=64  # TODO: refine these numbers when a case is available
+        export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS=10  # TODO: refine these numbers when a case is available
```

Comment:
Can we change this to a multiple of 6, e,g, 12? It gives developers option to try writing restart files on write grid comp?

Reply:
As seen from the variable name, it is per tile. So, it gets multiplied by 6 before ending up in `model_configure`.

Reply:
@junwang-noaa The meaning of the variable has already been changed to be per tile (we multiply by 6 later), so it is impossible for the end value to not be divisible by 6.

Reply:
Thanks for the explanation.